patches/chrome-browser-ui-webui-flags_ui.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/webui/flags_ui.cc b/chrome/browser/ui/webui/flags_ui.cc
+index 6bce6f8dd26ba64a131d6efadd30c8964db6d6eb..084d41a41d43fcf14dfe3ff8c128095ce571a645 100644
+--- a/chrome/browser/ui/webui/flags_ui.cc
++++ b/chrome/browser/ui/webui/flags_ui.cc
+@@ -57,7 +57,7 @@ content::WebUIDataSource* CreateFlagsUIHTMLSource() {
+   content::WebUIDataSource* source =
+       content::WebUIDataSource::Create(chrome::kChromeUIFlagsHost);
+   source->OverrideContentSecurityPolicyScriptSrc(
+-      "script-src chrome://resources 'self' 'unsafe-eval';");
++      "script-src chrome://resources chrome://brave-resources 'self' 'unsafe-eval';");
```

Comment:
W/o this, I got 
```
"Refused to load the script 'chrome://brave-resources/page_specific/flags/flags_overrides.js' because it violates the following Content Security Policy directive: "script-src chrome://resources chrome://brave-resource 'self' 'unsafe-eval'". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.", source: chrome://flags/
```

Reply:
we can't keep adding this everywhere, we need to find another way to handle this, maybe in `OverrideContentSecurityPolicyScriptSrc`

Reply:
Most of other webui pages already have `chrome://brave-resources` by https://github.com/brave/brave-core/blob/master/chromium_src/chrome/browser/ui/webui/webui_util.cc#L30.
but flags ui doesn't call `webui::SetupWebUIDataSource`.
If we add it in `OverrideContentSecurityPolicyScriptSrc`, it could overwrite different csp.
I think we should patch here for flags ui. WDYT?

Reply:
I think this is a fairly low risk change, given the page it's on. ++ from me personally

Reply:
IMO, patching is more proper in this case because we could miss upstream change if we replace it.

Reply:
no matter what this is the wrong patch because it's not extensible and it will require manual intervention if the upstream csp changes. We already have one patch for URLDataSource and a chromium_src override for webui_util and both of them are problematic because for the same reasons. The webui_util override and the URLDataSource patches needs to go away and be replaced by a single patch for `URLDataManagerBackend::GetHeaders` before `headers->AddHeader(base);` that allows us to make whatever changes we want to the csp (in this case inserting `chrome://brave-resources` after any `chrome://resources` entries